tests/unit/llm/test_anyscale.py

```diff
@@ -12,39 +12,51 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import importlib
 import os
+from typing import Generator
+from unittest import mock
 
-from distilabel.llm.anyscale import AnyscaleLLM
+import pytest
+from distilabel.llm import anyscale
 
 
+@pytest.fixture
+def reload_anyscale_module() -> Generator[None, None, None]:
+    importlib.reload(anyscale)
+    yield
```

Comment:
Again, I get this, but will be bad for UX, as the value can be set after the import and it won't work